src/app/feature/calendar/dialogs/card-details-dialog/card-details-dialog.component.ts

```diff
@@ -8,8 +8,12 @@ import {SafePipe} from '../../pipes/safe.pipe';
 
 @Component({
   selector: 'xmas-card-details-dialog',
-  standalone: true,
-  imports: [CommonModule, CalendarCardFactoryDirective, SafePipe, SocialProfileUrlPipe],
+  imports: [
+    CommonModule,
```

Comment:
Ideally we should import the `CommonModule` but own the directives we use. In this case it looks like we're only pulling `NgTemplateOutletDirective`

Reply:
I think we should read "should not import the CommonModule but only the directives we use", and I agree 👍